crud.lua

```diff
@@ -23,6 +23,8 @@ local stats = require('crud.stats')
 
 local crud = {}
 
+crud['_VERSION'] = require('crud.version')
```

Comment:
Let's use the same dotted notation as everything below and add a simple luadoc description like in others.

Reply:
Done